pkg/scheduler/reconciliation/reconciliation_test.go

```diff
@@ -729,3 +681,131 @@ func dbConnection(t *testing.T) db.Connection {
 	}
 	return dbConn
 }
+
+func TestReconciliationParallel(t *testing.T) {
+
+	t.Run("Create multiple instances of single reconciliations in parallel", func(t *testing.T) {
```

Comment:
As this test case  and 'Update single operation state in multiple parallel threads'  and 'Mark single reconciliation as finished in multiple parallel threads' are having the same body, it would be better to use one method for it and pass the changing code as closure variable to it.

Reply:
Done